Corgibytes.Freshli.Cli/CommandRunners/CacheCommandRunner.cs

```diff
@@ -33,5 +34,37 @@ public override int Run(CachePrepareCommandOptions options)
             return success ? 0 : 1;
         }
     }
-}
 
+    public class CacheDestroyCommandRunner : CommandRunner<CacheDestroyCommandOptions>
+    {
+        public CacheDestroyCommandRunner(IServiceProvider serviceProvider, Runner runner)
+            : base(serviceProvider, runner)
+        {
+
+        }
+
+        public override int Run(CacheDestroyCommandOptions options)
+        {
+            // Skip prompt if the --force flag is passed
+            if (options.Force)
+                goto Destroy;
+
+            // Prompt the user whether they want to destroy the cache
+            Console.Out.Write($"Do you want to destroy the Freshli cache at {options.CacheDir.ToString()}? [y/N] ");
+            string choice = Console.In.ReadLine();
+            string[] choicesToProceed = {"y", "Y"};
+            // If a "proceed" choice is not input, abort operation.
+            if (choicesToProceed.Contains(choice) == false)
+            {
+                Console.Out.WriteLine("Operation aborted. Cache not destroyed.");
+                return 0;
+            }
+
+            // Destroy the cache
+            Destroy:
+            Console.Out.WriteLine("Destroying cache...");
+            bool success = Cache.Destroy(options.CacheDir);
+            return (success == true ? 0 : 1);
```

Comment:
The parentheses and `== true` can be removed here.

```suggestion
            return success ? 0 : 1;
```

Also, since this is the second place that we're translating a boolean to an exit code, I think it makes sense to create an extension method on `System.Boolean` that does just that. Something like this.

```csharp
public static class BooleanExtensions
{
    public static int ToExitCode(this bool value)
    {
        return value ? 0 : 1;
    }
}
```

And then the code in this location would change to:

```csharp
return Cache.Destroy(options.CacheDir).ToExitCode();
```